eclass/kernel-install.eclass

```diff
@@ -38,13 +45,92 @@ esac
 
 inherit dist-kernel-utils mount-boot toolchain-funcs
 
+LICENSE="GPL-2"
+if [[ ${KERNEL_IUSE_GENERIC_UKI} ]]; then
+	# Keep in sync with the dracut modules included in our prebuilt images!
+	#
+	# Apache-2.0 for dev-libs/openssl, app-crypt/argon2
+	_INITRD_LICENSE=" Apache-2.0 "
```

Comment:
We may want to use an array instead.

Also, I wonder if this violates https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0106, but that rule doesn't really work for eclasses anyway, so...

Reply:
Sorry, I see you had this discussion already at https://github.com/gentoo/gentoo/pull/34024#discussion_r1430700286.